content/applications/general/email_communication/email_servers.rst

```diff
@@ -197,11 +197,8 @@ alias in your mail server.
 Set up different dedicated servers for transactional and mass mails
 ===================================================================
 
-Odoo is subject to a :ref:`daily email limit <email_communication/daily_limit_mail>` to prevent abuse. 
-However, if needed, you can use a separate Mail Transfer Agent (MTA) servers for transactional
-e-mails and mass mailings.
-Example: use Odoo's own mail server for transactional e-mails, and Sendgrid, Amazon SES, or Mailgun
-for mass mailings. Another alternative is to use Postmark for transactional e-mails, and Amazon SES
+You can use a separate Mail Transfer Agent (MTA) servers for transactional e-mails and mass mailings.
```

Comment:
Hi Jonathan, thanks for submitting this change. I am just going to change some of the wording, everything else looks great. 

``In Odoo a separate Mail Transfer Agent (MTA) server can be used for transactional e-mails and mass mailings.``